server/schedule/filter/filters.go

```diff
@@ -31,25 +31,25 @@ import (
 )
 
 var (
-	statusOK                 = plan.NewStatus(plan.StatusOK)
-	statusNoNeed             = plan.NewStatus(plan.StatusNoNeed)
-	statusDown               = plan.NewStatus(plan.StatusStoreUnavailable, "store is lost for longer than 'max-store-down-time' setting")
-	statusTombstone          = plan.NewStatus(plan.StatusStoreUnavailable, "store is tombstone")
-	statusDisconnected       = plan.NewStatus(plan.StatusStoreUnavailable, "store is lost for more than 20s")
-	statusBusy               = plan.NewStatus(plan.StatusStoreUnavailable, "store is busy")
-	statusOffline            = plan.NewStatus(plan.StatusStoreDraining, "store is in the process of offline")
-	statusLowSpace           = plan.NewStatus(plan.StatusStoreLowSpace, "store space is not enough, please scale out or change 'low-space-ratio' setting")
-	statusExcluded           = plan.NewStatus(plan.StatusStoreExcluded, "there has already had a peer or the peer is unhealthy in the store")
-	statusIsolation          = plan.NewStatus(plan.StatusIsolationNotMatch)
-	statusTooManySnapshot    = plan.NewStatus(plan.StatusStoreThrottled, "store snapshot has been piled up, the related setting is 'max-snapshot-count'")
-	statusTooManyPendingPeer = plan.NewStatus(plan.StatusStoreThrottled, "store has too many pending peers, the related setting is 'max-pending-peer-count'")
-	statusAddLimit           = plan.NewStatus(plan.StatusStoreThrottled, "store's add limit is exhausted, please check the setting of 'store limit'")
-	statusRemoveLimit        = plan.NewStatus(plan.StatusStoreThrottled, "store's remove limit is exhausted, please check the setting of 'store limit'")
-	statusLabel              = plan.NewStatus(plan.StatusLabelNotMatch)
-	statusRule               = plan.NewStatus(plan.StatusRuleNotMatch)
-	statusPauseLeader        = plan.NewStatus(plan.StatusStoreBlocked, "the store is not allowed to transfer leader, there might be an evict-leader-scheduler")
-	statusRejectLeader       = plan.NewStatus(plan.StatusStoreBlocked, "the store is not allowed to transfer leader, please check 'label-property'")
-	statusSlowStore          = plan.NewStatus(plan.StatusStoreBlocked, "the store is slow and are evicting leaders, there might be an evict-slow-store-scheduler")
+	statusStoreOK                 = plan.NewStatus(plan.StatusOK)
```

Comment:
Could we merge `statusStoreOK` with `statusRegionOK`?
How about defining a single file `status.go` to store these status variables?

Reply:
How about put all `status` into that file?